File: 1894-merge-strings-alternately/1894-merge-strings-alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        len1 = len(word1)
        len2 = len(word2)
        max_len = max(len1, len2)

        output = ""
        for i in range(max_len):
            if i < len1:
                output += word1[i]
            if i < len2:
                output += word2[i]

        return output

File: 1894-merge-strings-alternately/1894-merge-strings-alternately.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        string merged_word;
        int i;

        while (i < word1.length() && i < word2.length()){
            merged_word += word1[i];
            merged_word += word2[i];
            i++;
        }

        while (i < word1.length()){
            merged_word += word1[i];
            i++;
        }

        while (i < word2.length()){
            merged_word += word2[i];
            i++;
        }

        return merged_word;
    }
};